Tests/SourceKitLSPTests/HoverTests.swift

```diff
@@ -32,7 +32,9 @@ final class HoverTests: XCTestCase {
         This is a doc comment for S.
 
         Details.
-        """
+        """,
+      expectedRange:
+        .init(line: 3, utf16index: 7) ..< .init(line: 3, utf16index: 9)
```

Comment:
Could you run swift-format on your changes? https://github.com/swiftlang/sourcekit-lsp/blob/main/CONTRIBUTING.md#formatting

Reply:
done